source/JsMaterialX/JsMaterialXView/src/index.js

```diff
@@ -186,7 +197,7 @@ function init() {
         });
         obj.traverse((child) => {
             if (child.isMesh) {
-              generateTangents(child.geometry);
+              child.geometry.computeTangents();
```

Comment:
Oh, why did we do it ourselves before?

Reply:
that function only works if there is an index buffer

Reply:
Ok, so we can now assume that there's always such a buffer with the new file loader?